swift_browser_ui_frontend/src/common/router.js

```diff
@@ -1,15 +1,17 @@
-import Vue from "vue";
-import Router from "vue-router";
+import { createRouter, createWebHistory } from "vue-router";
 import FoldersView from "@/views/Folders.vue";
 import ObjectsView from "@/views/Objects.vue";
 import SharedObjects from "@/views/SharedObjects.vue";
 import TokensView from "@/views/Tokens.vue";
 
-Vue.use(Router);
-
-export default new Router({
-  mode: "history",
+export default createRouter({
+  history: createWebHistory(),
   routes: [
+    {
+      path: "/browse",
+      name: "AllFolders",
+      component: FoldersView,
+    },
```

Comment:
Do we really need this path? It's also related to my previous comment about router.replace 
I never get to this `/browse` actually :thinking:

Reply:
Do you think it should be removed?

Reply:
When I tested from my side without this part, I think it didn't affect anything so probably it could be removed.